lib/plugins/aws/package/lib/stripNullPropsFromTemplateResources.js

```diff
@@ -1,16 +1,22 @@
 'use strict';
 
+const stripNullPropsFromObj = (obj) => {
+  Object.entries(obj).forEach(([propName, propVal]) => {
+    if (propVal === null) {
+      delete obj[propName];
+    } else if (propVal && typeof propVal === 'object') {
```

Comment:
We can simplify to `if (typeof === 'object')`

Reply:
Good catch! Pushed out an update 👍